source/CommonJobs/CommonJobs.Application.EvalForm/Commands/GetEvaluationCalifications.cs

```diff
@@ -49,42 +49,113 @@ public override CalificationsDto ExecuteWithResult()
+        {
+            return new CalificationsDto()
+            {
+                View = UserView.Evaluation,
+                Evaluation = evaluationDto,
+                Califications = califications.Where(c => _loggedUser == c.EvaluatorEmployee || c.Owner == CalificationType.Auto).ToList()
+            };
+        }
+
+        private CalificationsDto GetEvaluation(CalificationsEvaluationDto evaluationDto, List<EvaluationCalification> califications)
+        {
+            return new CalificationsDto()
+            {
+                View = califications.Any(c => c.Owner == CalificationType.Company) ? UserView.Company : UserView.Responsible,
+                Evaluation = evaluationDto,
+                Califications = califications
+            };
+        }
+
+        private CalificationsDto GetAutoEvaluation(CalificationsEvaluationDto evaluationDto, List<EvaluationCalification> califications)
+        {
+            if (evaluationDto.ReadyForDevolution)
             {
                 return new CalificationsDto()
                 {
-                    View = califications.Any(c => c.Owner == CalificationType.Company) ? UserView.Company : UserView.Responsible,
+                    View = UserView.Auto,
                     Evaluation = evaluationDto,
-                    Califications = califications
+                    Califications = califications.Where(c => _evaluatedUser == c.EvaluatedEmployee && (c.Owner == CalificationType.Auto || c.Owner == CalificationType.Company)).ToList()
                 };
             }
-            else // Evaluator
+            else
             {
                 return new CalificationsDto()
                 {
-                    View = UserView.Evaluation,
+                    View = UserView.Auto,
                     Evaluation = evaluationDto,
-                    Califications = califications.Where(c => _loggedUser == c.EvaluatorEmployee || c.Owner == CalificationType.Auto).ToList()
+                    Califications = new List<EvaluationCalification>() { califications.Single(c => _evaluatedUser == c.EvaluatedEmployee && c.Owner == CalificationType.Auto) }
                 };
             }
         }
+
+        private CalificationsDto GetFinishedEvaluation(CalificationsEvaluationDto evaluationDto, List<EvaluationCalification> califications)
+        {
+            return new CalificationsDto()
+            {
+                View = (_loggedUser == evaluationDto.ResponsibleId)
+                ? UserView.Company
+                    : (_loggedUser == _evaluatedUser)
+                        ? UserView.Auto
+                        : UserView.Evaluation,
+                Evaluation = evaluationDto,
+                Califications = califications.Where(c => _evaluatedUser == c.EvaluatedEmployee && (c.Owner == CalificationType.Auto || c.Owner == CalificationType.Company)).ToList()
```

Comment:
So, when the evaluation is finished individual califications of each calificator are not accessible anymore?